src/TuDa.CIMS.Web/Components/ShoppingSearch.razor

```diff
@@ -0,0 +1,16 @@
+﻿<MudGrid Justify="Justify.Center">
+    <MudItem xs="12" sm="8" md="8">
+        <MudAutocomplete
+            @ref="_autocomplete"
+            Clearable="@true"
+            AdornmentIcon="@Icons.Material.Filled.Search"
+            AdornmentColor="@Color.Primary"
+            Label="Items"
+            SearchFunc="@Search"
+            ToStringFunc="@ToString"
+            Variant="Variant.Filled"
+            Dense=true
+            ShowProgressIndicator="true">
+        </MudAutocomplete>
```

Comment:
🔧 Add back in the `ValueChanged`. I found out what were wrong, you need to add the Type back in and then he takes a normal function as parameter.
```suggestion
        <MudAutocomplete
            T="AssetItem"
            @ref="_autocomplete"
            Clearable="@true"
            AdornmentIcon="@Icons.Material.Filled.Search"
            AdornmentColor="@Color.Primary"
            Label="Items"
            SearchFunc="@Search"
            ToStringFunc="@ToString"
            ValueChanged="@AssetItemSelectedInternal"
            Variant="Variant.Filled"
            Dense=true
            ShowProgressIndicator="true">
        </MudAutocomplete>
```